profiles/tests/test_profiles_api.py

```diff
@@ -47,3 +47,23 @@ def test_put_update_own_profile(user_api_client, profile):
 
     profile.refresh_from_db()
     assert profile.phone == phone_number_data['phone']
+
+
+def test_expected_profile_data_fields(user_api_client, profile):
+    expected_fields = (
+        'nickname',
+        'email',
+        'phone',
+        'language',
+        'contact_method',
+        'concepts_of_interest',
+        'divisions_of_interest',
+        'preferences'
+    )
+
+    user_profile_url = get_user_profile_url(profile)
+    profile_endpoint_data = get(user_api_client, user_profile_url)
+
+    assert len(profile_endpoint_data) == len(expected_fields)
+    for field_name in expected_fields:
+        assert field_name in profile_endpoint_data
```

Comment:
Using `set`s would be more pro here, ie. `set(profile_endpoint_data.keys()) == set(expected_fields)` (or just `== expected_fields` if `expected_fields` was a `set` to begin with, which would be super pro).

Reply:
Fantastisch! 👍

Reply:
Feel so much more like a pro already! 😎